# exception

def m1():
    try:
       return 10
    except:
       return 20    
    else:
       return 30
    finally:
       return 40 

print(m1())



